deflatebench.py

```
@@ -141,46 +116,70 @@ def printinfo():
     print(f"Runs: {str(cfgRuns['runs']):10} Trim worst: {str(cfgRuns['trimworst']):10}")
     print("")
 
-def printreport(comp,decomp,totals):
+def print_resultline(header,pct,comp,decomp,size=None):
+    printnn(f" {header:5}")
+    if isinstance(pct, float):
+        printnn(f"{pct:>7.3f}% ")
+    else:
+        printnn(f"{pct:>8} ")
+    if benchmode != 'decomp':
```

Ruff (0.14.14)

[error] Undefined name `benchmode`

(F821)